plugins/www/Optimove.js

```diff
@@ -1,39 +1,188 @@
+var exec = require("cordova/exec");
 
+var inAppInboxUpdatedHandler;
```

Comment:
is there a reason why this one is special? Why it doesnt reside on config like all others? 

I think in kumulos sdk the reason was that we wanted to be able to set it to null, but could not modify after config passed into initialisation. Here handlers are not part of config

Reply:
done